crates/cast/tests/e2e/script/declare.rs

```diff
@@ -21,7 +21,7 @@ async fn test_missing_field() {
         .args(args);
```

Comment:
The `e2e` tests with `--wait-timeout`, `--wait-retry-interval` flags (passed from CLI and parsed from Scarb.toml) are needed.

Reply:
yes, will be added in https://github.com/foundry-rs/starknet-foundry/issues/1201 , after we manage to add tests with integration network.

for now I have tested this manually, since this is virtaully not testable with devnet atm

Reply:
No proof the `PropertyFromCastConfig` approach works correctly when parsing stuff from Scarb.toml